llmake.py

```diff
@@ -1,5 +1,5 @@
 ###
-# Copyright 2024 Massimo Fioravanti
+# Copyright 2024 Massimo Fioravanti, Attilio Polito, Raffaele Russo
```

Comment:
i am not 100% sure but it should be that since the file already existed, the copyright does not get transferred due to additions, the copyright for the single changes is implied by the commit. So in this case the two added names should not be there.